Makefile

```diff
@@ -17,9 +17,14 @@ init-db: ## Create and setup the database
 	docker-compose run --rm \
 		php bash -ci './bin/console doctrine:database:create --if-not-exists && ./bin/console doctrine:schema:update --force'
 
+go-install:
```

Comment:
IMO, you're going to lose this installation every time you restart the docker.
You should be able to create a `Dockerfile` that extends the base image, with this *global* installation.

Something like 
```
// in Docker/go/DockerFile
FROM golang:1.12.5
RUN go get -u -v github.com/githubnemo/CompileDaemon
```

then

```
// in docker-compose.yml
  advisor:
    build: Docker/go
```

Reply:
Yes you are right ! It fixed !